tests/nextflow.config

```diff
@@ -0,0 +1,5 @@
+/*
+========================================================================================
+    Nextflow config file for running tests
+========================================================================================
+*/
```

Comment:
do we need this empty file?

Reply:
Honestly, No. This file is just to let user know they can override some settings here.nf-test is using the main nextflow.config.

Reply:
i ll remove this one in a future PR